### ISSUE TITLE
"recipe failed" when building on Ubuntu

### ISSUE THREAD

**Issue by ashes999** [IronLanguages/ironpython3#294]
Hi,

I tried the following to build the code:

- `git clone [...] --recurse`
- `make`

I get an error, with very little information:

```
Configuration: Debug Platform: AnyCPU
		Configuration: Debug Platform: AnyCPU
Makefile:4: recipe for target 'debug' failed
make: *** [debug] Error 1
```

What exactly does this mean? I checked the `Makefile` and ran line 4 (`xbuild Build.proj /t:Build /p:Mono=true /p:BuildFlavour=Debug /p:Platform="Any CPU" /verbosity:minimal /nologo`) manually, but it just prints the same first two lines (configuration: Debug ...) and nothing apparently happens.

**Comment by slide** [comment]
Did you do a submodule initialization for the DLR sources?

**Comment by ashes999** [comment]
Yes. I believe `git checkout [...] --recurse` included the submodules, because without this, I got a build failure about something missing from the DLR directory (which was empty).

**Comment by slide** [comment]
What version of mono do you have installed?

**Comment by ashes999** [comment]
`mono --version` tells me `Mono JIT compiler version 4.2.1 (Debian 4.2.1.102+dfsg2-7ubuntu4)`

**Comment by slide** [comment]
I believe you need at least 4.6 for IronPython

**Comment by ashes999** [comment]
Looks like you're right. I can build the proposed fix after upgrading to Mono 4.8.

**Comment by slide** [comment]
We need to update our docs to specify the minimum Mono version, so I'll leave this open.
